test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       hostNetwork: true
       containers:
```

Comment:
I think this line: https://github.com/kubernetes/kubernetes/blob/519eee70ed7fbdaa41dc730c644ca019e508ab86/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml#L14

might need to get changed to `csi-node-sa` based on the other changes

Reply:
Fixed.